temporalio/workflow.py

```diff
@@ -1051,6 +1063,34 @@ def get_current_details() -> str:
     return _Runtime.current().workflow_get_current_details()
 
 
+def has_last_completion_result() -> bool:
+    """Get the last completion result of the workflow. This be None if there was
+    no previous completion or the result was None
+    """
+    return _Runtime.current().workflow_has_last_completion_result()
+
+
+@overload
+def get_last_completion_result() -> Optional[Any]: ...
+
+
+@overload
+def get_last_completion_result(type_hint: Type[ParamType]) -> Optional[ParamType]: ...
+
+
+def get_last_completion_result(type_hint: Optional[Type] = None) -> Optional[Any]:
+    """Get the last completion result of the workflow. This be None if there was
+    no previous completion or the result was None. has_last_completion_result()
+    can be used to differentiate.
```

Comment:
[Same typos causing grammatical incorrectness as above]

I'd like us to explain what a "completion result" is, and therefore what a "last completion result" is.